src/main/java/dk/bec/bookanything/mapper/ReservationMapper.java

```diff
@@ -0,0 +1,40 @@
+package dk.bec.bookanything.mapper;
+
+import dk.bec.bookanything.dto.ReservationCreateDto;
+import dk.bec.bookanything.dto.ReservationReadDto;
+import dk.bec.bookanything.model.ReservationEntity;
+import dk.bec.bookanything.service.BookableObjectService;
+import dk.bec.bookanything.service.UserService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReservationMapper {
+
+    private final UserService userService;
+    private final BookableObjectService bookableObjectService;
+
+
+    public ReservationMapper(UserService userService, BookableObjectService bookableObjectService) {
+        this.userService = userService;
+        this.bookableObjectService = bookableObjectService;
+    }
+
+    public ReservationEntity mapReservationCreateDtoToReservationEntity(ReservationCreateDto reservationCreateDto, Long id){
+        return ReservationEntity.builder()
+                .id(id)
+                .dateFrom(reservationCreateDto.getDateFrom())
+                .dateTo(reservationCreateDto.getDateTo())
+                .peopleNumber(reservationCreateDto.getPeopleNumber())
+                .userEntity(userService.getUserById(reservationCreateDto.getUserId()).get())
```

Comment:
Avoid calling .get() on Optional without checking its presence - could result in NPE.